std_msgs/Header header
geometry_msgs/Pose pose
float64 scale
geometry_msgs/Quaternion rotation_metric_to_dso
geometry_msgs/Pose imu_to_cam
